packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -99,6 +118,23 @@ const OTPInput = ({
     }
   }, [otpValue, otpLength, name, inputValue, onOTPFilled]);
 
+  useEffect(() => {
```

Comment:
Suggestion: add a comment to explain why we need this `useEffect`

Reply:
Done